doc.go

```diff
@@ -1,32 +1,70 @@
-// Package image provides libraries and commands to interact with containers images.
+// The package image provides libraries and commands to interact with container images.
 //
 //  package main
 //
 //  import (
-//   	"context"
-//  	"fmt"
+//	"context"
+//	"fmt"
 //
-//  	"github.com/containers/image/v5/docker"
+//	"github.com/containers/image/v5/docker"
 //  )
 //
 //  func main() {
-//  	ref, err := docker.ParseReference("//fedora")
-//  	if err != nil {
-//  		panic(err)
-//  	}
-//  	ctx := context.Background()
-//  	img, err := ref.NewImage(ctx, nil)
-//  	if err != nil {
-//  		panic(err)
-//  	}
-//  	defer img.Close()
-//  	b, _, err := img.Manifest(ctx)
-//  	if err != nil {
-//  		panic(err)
-//  	}
-//  	fmt.Printf("%s", string(b))
+//	ref, err := docker.ParseReference("//fedora")
+//	if err != nil {
+//		panic(err)
+//	}
+//	ctx := context.Background()
+//	img, err := ref.NewImage(ctx, nil)
+//	if err != nil {
+//		panic(err)
+//	}
+//	defer img.Close()
+//	b, _, err := img.Manifest(ctx)
+//	if err != nil {
+//		panic(err)
+//	}
+//	fmt.Printf("%s", string(b))
 //  }
 //
 //
+// ## Notes on running in rootless mode
+//
+// If your application shall run in rootless mode, for instance to be able to
+// read from podman's user storage, then the following additional steps have to
+// be performed at startup of your application:
```

Comment:
Something like “If your application needs to access a containers/storage store in rootless mode”, please; we don’t want the more common users of c/image/docker to do any of this.

Reply:
Makes sense, I've adjusted the wording accordingly.